dns.go

```diff
@@ -159,6 +169,41 @@ func initBackOffStrategy() internal.Backoff {
 	}
 }
 
+func resolveDnsServerAddress(log logr.Logger) string {
+	if inputConfig.useKubeDnsServer {
+		ip, err := findKubeDnsServerIp(log)
+		if err == nil {
+			return ip
+		}
```

Comment:
Can we print the error using log.Error or so?

Reply:
It was printed in findKubeDnsServerIp method. Let me bring that here so that it will just be one log statement.

Reply:
Done.